.github/pull_request_template.md

```diff
@@ -21,6 +21,27 @@
 
 <!-- screenshot (or gif/video) that demonstrates the functionality, specially important if it's a bug fix. -->
 
+<!-- Uncomment this section for status-go upgrade/dogfooding pull requests
+
+### Impact on end user
+
+What is the impact of these changes on the end user (before/after behaviour)
+
+### How to test
+
+- How should one proceed with testing this PR.
+- What kind of user flows should be checked?
+
+### Risk 
+
+Described potential risks and worst case scenarios.
+
+Tick **one**:
+- [ ] Low risk: 2 devs MUST perform testing as specified above and attach their results as comments to this PR **before** merging.
+- [ ] High risk: QA team MUST perform additional testing in the specified affected areas **before** merging.
```

Comment:
We can't guarantee the timing though; as it interferes with priorities for our platforms. so it can take a while or priority should be set by the platform lead respectively.

cc @anastasiyaig

Reply:
I can communicate from  Waku PoV how critical the changes are to help perform leads prioritize.

cc @chair28980